src/main/java/com/unosquare/admin_core/back_end/repository/DashboardRepository.java

```diff
@@ -23,6 +23,20 @@
                                                    @Param("startDate") LocalDate startDate,
                                                    @Param("endDate") LocalDate endDate);
 
+    @Query(value = "SELECT e FROM Event e " +
+            "WHERE " +
+            "((e.startDate BETWEEN :startDate AND :endDate) OR " +
+            "(e.endDate BETWEEN :startDate AND :endDate) OR " +
+            "(e.startDate > :startDate AND e.endDate < :endDate)) " +
+            "AND " +
+            "(e.eventStatus = '1') AND (e.eventType = '1') OR (e.eventType = '2') " +
```

Comment:
Can you use the enum here?

Reply:
If you get a chance could you show me the enum in this?